basic-examples/bind-constants/README.md

```diff
@@ -0,0 +1,75 @@
+# Binding Repeated Values
+
+## Workflow Description
+
+This workflow demonstrates how to use the Arcaflow builtin function `bindConstants()` to repeat a set of input values with each item in your input list.
+
+## Files
+
+- [`workflow.yaml`](workflow.yaml) -- Defines the workflow input schema, the plugins to run
+  and their data relationships, and the output to present to the user
+- [`input.yaml`](input.yaml) -- The input parameters that the user provides for running
+  the workflow
+- [`config.yaml`](config.yaml) -- Global config parameters that are passed to the Arcaflow
+  engine

```

Comment:
We're missing the `subworkflow.yaml` entry, here.

Reply:
did it